enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1986,12 +1986,12 @@ func (cdcw *cdcWriter) writeRawChunk(fileRecord *rfpb.FileRecord, key filestore.
 	if err != nil {
 		return status.InternalErrorf("failed to write raw chunk: %s", err)
 	}
-	if err := wcm.Close(); err != nil {
-		return status.InternalErrorf("failed to close while writing raw chunk: %s", err)
-	}
 	if err := wcm.Commit(); err != nil {
 		return status.InternalErrorf("failed to commit while writing raw chunk: %s", err)
 	}
+	if err := wcm.Close(); err != nil {
```

Comment:
Can you put the Close in a defer so that it's called even if the write or commit fail?

Reply:
Done